sources/packages/backend/apps/api/src/auth/roles.enum.ts

```diff
@@ -10,6 +10,7 @@ export enum Role {
   AESTCreateInstitution = "aest-create-institution",
   AESTEditCASSupplierInfo = "aest-edit-cas-supplier-info",
   AESTCASInvoicing = "aest-cas-invoicing",
+  AESTCASInvoiceBatchExpense = "aest-cas-expense-authority",
```

Comment:
Please adjust the enum role to reflect its value following the same idea from the other enum properties/values.
```
AESTCCASExpenseAuthority= "aest-cas-expense-authority",
```